README.md

```diff
@@ -26,6 +26,7 @@ Keybinding                              | Description
 <kbd>Ctrl+K</kbd>, <kbd>Shift+Tab</kbd> | Previous view.
 <kbd>Ctlr+J</kbd>, <kbd>Tab</kbd>       | Next view.
 <kbd>Ctrl+H</kbd>, <kbd>Alt+H</kbd>     | Toggle history.
+<kbd>Ctrl+E</kbd>                       | Export as cURL command
```

Comment:
Yeah, it's otherwise a good keybinding, that just popped out to me when I saw it (which may be due to being an Emacs user :P ).